### PR TITLE
Upgrade Twitch plugin to .net8

### PR DESCRIPTION
**Pull request by Oleksandr-Podranetskyy**

### CODE REVIEW

File: TwitchPlugin/TwitchPlugin/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "profiles": {
+    "TwitchPluginWin": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files\\Logi\\LogiPluginService\\LogiPluginService.exe"
+    },
+    "TwitchPluginMac": {
+      "commandName": "Executable",
+      "executablePath": "/Applications/Loupedeck.app/Contents/MacOS/Loupedeck"
```
Comment (syashchuk-dev) [inline TwitchPlugin/TwitchPlugin/Properties/launchSettings.json:9]: The path has changed. The app is now in `/Applications/Utilities/LogiPluginService.app`.

File: TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj [hunk 2]
```diff
@@ -1,328 +1,213 @@
...
+    <Compile Include="Models\TwitchPluginConfig.cs" />
+    <Compile Include="Models\EventArgs.cs" />
+    <Compile Include="Models\UserInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="TwitchApplication.cs" />
+    <Compile Include="TwitchPlugin.cs" />
+    <Compile Include="Proxy\TwitchProxy.Channel.cs" />
+    <Compile Include="Proxy\TwitchProxy.Connectivity.cs" />
+    <Compile Include="Proxy\TwitchProxy.cs" />
+    <None Include="Proxy\TwitchProxy.Chatters.cs" />
```
Comment (syashchuk-dev) [inline TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj:62]: Why do we include this file but don't compile?

File: TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj [hunk 3]
```diff
@@ -1,328 +1,213 @@
...
+    <Compile Include="Proxy\TwitchProxy.Viewers.cs" />
+  </ItemGroup>
+  <ItemGroup>
```
Comment (syashchuk-dev) [inline TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj:65]: Looks like we have a bunch of duplicates here.

**Review by syashchuk-dev — CHANGES_REQUESTED** [review]

File: TwitchPlugin/TwitchPlugin/Properties/AssemblyInfo.cs
[the remark this answers is not in the document]
Comment (Oleksandr-Podranetskyy) [reply TwitchPlugin/TwitchPlugin/Properties/AssemblyInfo.cs:1]: The file was removed

File: TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj (hunk 3 above)
Comment (Oleksandr-Podranetskyy) [reply TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj]: The duplicates were removed

**Review by syashchuk-dev — APPROVED** [review]

**Pull request merged**